src/common/eeprom.cpp

```diff
@@ -13,7 +13,7 @@
 #include "wdt.h"
 #include "../Marlin/src/module/temperature.h"
 
-static const constexpr uint8_t EEPROM__PADDING = 1;
+static const constexpr uint8_t EEPROM__PADDING = 4;
```

Comment:
too bad we can't make an empty member, right?

Reply:
too bad :_(